.github/workflows/build.yml

```
@@ -151,15 +151,26 @@ jobs:
     if: fromJSON(needs.check_source.outputs.run_tests)
     strategy:
```

If one fails, don't cancel the others:
```suggestion
    strategy:
      fail-fast: false
```